app/classes/observation_report/mycoflora.rb

```diff
@@ -107,5 +114,19 @@ def add_mycoflora_ids!(rows, col)
       )
       add_column!(rows, vals, col)
     end
+
+    def add_collector_ids!(rows, col)
+      vals = CollectionNumber.connection.select_rows %(
```

Comment:
Should a sort be added to this SQL statement? See  https://github.com/MushroomObserver/mushroom-observer/pull/391#issuecomment-364300163 and https://github.com/MushroomObserver/mushroom-observer/pull/391#issuecomment-364312470.

Reply:
Sure, if a sort is needed to derandomize the final result, I'm all in favor of it.  I hate functions that leave order random.  (The alternative is to sort the results before the assertion in the unit test.  Whatever works for you.)

Reply:
Hard (maybe impossible?) to tell if it's really needed. This is the only time the test failed.

Reply:
But I like the idea of sorting it in the test. Epsilon less of a performance hit in production.